apps/librelingo_types/librelingo_types/data_types.py

```diff
@@ -1,7 +1,63 @@
 from collections import namedtuple
 
+class TextToSpeechSettings(
+    namedtuple(
+        "TextToSpeechSettings",
+        [
+            "provider",
+            "voice",
+            "engine",
+        ],
+        defaults=["Polly", "Lupe", "standard"]
+    )
+):
+    """
+    Settings about how to use TTS to generate audios
+
+    ### Usage example:
+
+    ```python
+    TextToSpeechSettings(
+        provider="Polly",
+        voice="Aditi",
+        engine="standard"
+    )
+    ```
+    """
+
+    pass
+
+default_text_to_speech_settings = TextToSpeechSettings()
+
+class AudioSettings(
+    namedtuple(
+        "AudioSettings",
+        [
+            "enabled",
+            "text_to_speech_settings",
+        ],
+        defaults=[True, [default_text_to_speech_settings]]
+    )
+):
+    """
+    Settings for audio in a course
+
+    ### Usage example:
+
+    ```python
+    AudioSettings(
+        enabled=True,
+        text_to_speech_settings=[TextToSpeechSettings()]
+    )
+    ```
+    """
+
+    pass
+
+default_audio_settings = AudioSettings()
+
 
-class Settings(namedtuple("Settings", ["audio_files_enabled"], defaults=[True])):
+class Settings(namedtuple("Settings", ["audio_settings"], defaults=[default_audio_settings])):
```

Comment:
this is meant to be a breaking change, right?

Reply:
It is breaking in that the format of the `course.yaml` file changes, yes. Also, because I agree with your idea in the other comment about audio being off by default, it will be breaking in changing the default.